sdk/python/feast/feature_store.py

```diff
@@ -1223,19 +1233,25 @@ def _populate_odfv_dependencies(
         if len(grouped_odfv_refs) > 0:
             for odfv, _ in grouped_odfv_refs:
                 for fv in odfv.input_feature_views.values():
-                    table_join_keys = [
-                        entity_name_to_join_key_map[entity_name]
-                        for entity_name in fv.entities
+                    feature_deps = [
```

Comment:
Trying to follow this code I'm confused what `feature_deps` means here and what this block of code does.. Any possibility to rename/refactor/add comments?

Reply:
Had another pass at this and caught a case I hadn't considered + added some comments etc. More clear now?